.github/workflows/publish.yml

```diff
@@ -0,0 +1,94 @@
+# Publishes built jars to distribution platforms
+name: Publish
+
+on:
+  push:
+    tags:
+      - 'v*.*.*' # any semver tag, e.g. v1.2.3
+
+# if a second commit is pushed quickly after the first, cancel the first one's build
```

Comment:
I don't think this should be used here as for every tag we want to have build (release)